nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc

```diff
@@ -7,14 +7,13 @@ enable_antialiasing: true
 show_displays: true
 show_status_bar: true
 show_capture_tools: true
-window_width: 1920
-window_height: 1016
-view_scale: 0.564473748
-offset_x: -55.0469894
-offset_y: -4.65194941
+window_width: 1000
```

Comment:
Why does changing these values fix the problem?

Reply:
I think that what is making the system working under Humble is the remove of this line of code `image_transport: raw` I didn't try any possible edits.

Reply:
OK, lets just change that item then :-)